deploy/devops/tf-env/pub_lb_sec.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# shellcheck disable=SC2155
+export SEC_LIST_ID=$(oci network subnet get --subnet-id $PUB_LB_SUBNET_ID --query 'data."security-list-ids"[0]' --raw-output)
```

Comment:
@alcampag, why is the `kube-proxy` port `10256` not opened on the Terraform side? Is not the official OKE module supporting this SecList? I ask out of curiosity.

The alternative is to move to NSGs, as you mention in the comment, which sounds good as a next move.

Reply:
The way the new module works is that everything should be handled through NSGs, so even if it creates the LB_subnet and the relative security list, by default that security list is empty.

This forces the user to attach the related NSG through annotations at the Service level, by specifying the NSG OCID.  The kube-procy rule is in the NSG for the LB, but as I do not know beforehand the OCID of the NSG, I opted to add the security rule in the load balancer security list.

The OKE team also recently added an NSG mode for Load Balancers, so this could be further simplified.